src/main/java/pl/trainingapp/controller/HomeController.java

```diff
@@ -0,0 +1,15 @@
+package pl.trainingapp.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
```

Comment:
You can replace @Controller and @ResponseBody with @RestController

Reply:
Replacer @Controller with @RestController